website/versioned_docs/version-27.2/JestObjectAPI.md

```diff
@@ -578,6 +578,47 @@ Returns the `jest` object for chaining.
 
 Restores all mocks back to their original value. Equivalent to calling [`.mockRestore()`](MockFunctionAPI.md#mockfnmockrestore) on every mocked function. Beware that `jest.restoreAllMocks()` only works when the mock was created with `jest.spyOn`; other mocks will require you to manually restore them.
 
+### `jest.mocked<T>(item: T, deep = false)`
```

Comment:
I'll roll a new version of the docs after merging and releasing this PR, so please remove this change 🙂

Reply:
sure 👍